investment-app/TRINV.Api/TRINV.Domain/Entities/Investment.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace TRINV.Domain.Entities;
+
+using System.ComponentModel.DataAnnotations;
+using Enums;
+using static Validations.EntityValidationConstants.Investment;
+
+public class Investment
+{
+    public Investment()
+    {
+        IsFromOutsideProvider = false;
```

Comment:
Defaul is false, You don't need it. Remove and  the constructor

Reply:
Done.